tests/braille-specs/hbo.yaml

```diff
@@ -24,3 +46,36 @@ tests:
   - ["בְּכָל־לְבָבְךָ֥", "⠃⠄⠡⠣⠇⠤⠇⠄⠧⠣⠧⠄⠡⠣⠨⠍"]
   - ["וּבְכָל־נַפְשְׁךָ֖", "⠥⠧⠄⠡⠣⠇⠤⠝⠉⠋⠄⠩⠄⠡⠣⠨⠞"]
   - ["וּבְכָל־מְאֹדֶֽךָ׃", "⠥⠧⠄⠡⠣⠇⠤⠍⠄⠁⠸⠪⠙⠑⠈⠡⠣⠲"]
+
+#tests2. This test attempts to show how the current Hebrew table does not
+# accurately produce the desired braille for the consanant, vowel dagesh. 
+  - ["בָּרָא", "⠃⠣⠗⠣⠁", xfail: true]
+  - ["אֱלֹהִים ", "⠁⠢⠇⠕⠓⠔⠍ ", xfail: true]
+  - ["הָאָרֶץ.", "⠓⠣⠁⠣⠗⠑⠋⠮", xfail: true]
+  - ["הָיְתָה תֹה. ", "⠓⠣⠚⠄⠹⠣⠓ ⠹⠕⠓⠲ ", xfail: true]
+  - ["וְהָאָרֶץ ‬", "⠺⠄⠓⠣⠁⠣⠗⠑⠮ ‬", xfail: true]
```

Comment:
What is this thing about hidden Unicode characters ("This file contains bidirectional Unicode text that may be interpreted or compiled differently than what appears below. To review, open the file in an editor that reveals hidden Unicode characters", [learn more](https://github.blog/changelog/2021-10-31-warning-about-bidirectional-unicode-text/)). Apparently the problem is the U+202C character.

Reply:
Yes, I noticed that too, and when I deleted it, both in the input and output, I got an unexpected pass.